src/test/java/PlayersTest.java

```diff
@@ -0,0 +1,39 @@
+import domain.Name;
+import domain.Player;
+import domain.Players;
+import domain.Position;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class PlayersTest {
+
+  @Test
+  void 플레이어리스트를_생성_한다() {
+    Player testPlayer1 = new Player(new Name("pobi"), new Position(0));
```

Comment:
다음과 같은 코드와는 어떤 차이점이 있을까요? 민주님은 어떤 코드가 더 좋다고 생각하시는지 궁금합니다 :)

```suggestion
    Player testPlayer1 = new Player("pobi", 0);
```

제 의견이 무조건 정답은 아니니 편하게 말씀해주세요 😄

Reply:
`Name` 클래스와 `Postion` 클래스를 먼저 TDD 에 맞춰서 작성하고 그 다음으로 `Player` 클래스를 작성했었는데 ,  '자료형이 Name 이니까 생성자에서는 Name으로 받아야 겠지'  같은 마음으로 작성했던 부분입니다. 제안해주신 방식으로 작성하면 객체를 선언할 때 더 간결해져 읽기가 쉬워지므로 좋다고 느꼈습니다. 이에 Player 생성자 부분을 수정했습니다.

리뷰를 참고하면서 아직 객체지향에 익숙하지 않은점이 있다고 느꼈습니다. 익숙하지 않다보니 어떤것이 어떻게까지 표현이 가능하구나 같은 유연함이 부족한것 같습니다 ㅎㅎ